### DISCUSSION TITLE
Air Copy feature

### DISCUSSION THREAD

**Discussion by bogdanstk** [egzumer/uv-k5-firmware-custom#439] [Ideas]
I can't seem to find an Air Copy feature (i.e. copy the channel list from one radio to another) that is supported by the OE firmware. Is this not implemented or it is just me that can't find it? I read the documentation and can't find any clues regarding this.

For me it seems like a useful feature when out on the field.

**Comment by OE1MWW** [comment]
AFIK this feature has been removed to make room for other option - so - it does not / can not exist anymore.

**Comment by JuantAldea** [comment]
It can be enabled at compile time. Take a look at the Makefile.

**Comment by PolyCatDev** [comment]
How do you use this feature? I've enabled it at compile time but I can't find any documentation online on how to use it. I've found some yt videos saying to power on with PTT and side btn 2 pressed but I get nothing.

**Reply by PolyCatDev** [reply]
Update: Last night it didn't work, I tried it today and it worked for some reason. I think it wasn't working bacuse I was in VFO mode when trying to enter Air copy.
